src/applications/proxy-rewrite/tests/redirects/index.unit.spec.js

```diff
@@ -29,6 +30,37 @@ describe('Redirect replaced pages', () => {
 
     expect(fakeWindow.location.href).to.be.undefined;
   });
+
+  it('should redirect absolute redirects over catch-all redirects', () => {
+    const fakeWindow = {
+      location: {
+        host: 'www.altoona.va.gov',
+        pathname: '/locations/directions.asp',
```

Comment:
Let's remove this 1:1 redirect, we do not need to redirect the legacy directions page.